src/util/expr_util.cpp

```diff
@@ -7,6 +7,7 @@ Author: Daniel Kroening, kroening@kroening.com
 \*******************************************************************/
 
 
+#include <unordered_set>
```

Comment:
Nit pick: please move to bottom of the include list, the header file corresponding to this implementation should come first.